napari/_vispy/vendored/volume.py

```diff
@@ -7,19 +7,19 @@
 --------------------
 
 In Python, we define the six faces of a cuboid to draw, as well as
-texture cooridnates corresponding with the vertices of the cuboid. 
+texture cooridnates corresponding with the vertices of the cuboid.
 The back faces of the cuboid are drawn (and front faces are culled)
-because only the back faces are visible when the camera is inside the 
+because only the back faces are visible when the camera is inside the
```

Comment:
are these extra spaces there in the vispy shader? good to try to keep things in sync there - no major issue though

Reply:
The spaces were removed by my vispy PR already, so now that it's merged everything should be in sync!